src/risk_distributions/risk_distributions.py

```diff
@@ -591,6 +595,27 @@ def get_parameters(
 
         return weights, params
 
+    @staticmethod
+    def fill_missing_weights(weights: Parameters, expected_columns) -> Parameters:
+        weights = copy.deepcopy(weights)
```

Comment:
Why do we need to deepcopy if we're just reassigning this to the variable containing the input `weights` in the only place this method is being used?

Reply:
I understand that, but you're just assigning the copy to the variable housing the original weights, so why not modify it in place?